Resources/public/scss/modules/content-tree/_main.scss

```diff
@@ -41,4 +41,10 @@
         color: $ez-color-base-light;
         cursor: pointer;
     }
+
+    &__no-items-message {
+        padding: calculateRem(10px);
+        font-size: calculateRem(20px);
+        color: $ez-color-base-light;
+    }
```

Comment:
- For this kind of messages we are using `color: $ez-color-base-dark;`.
- Font-size, please change it to 12px (the rule we are following is that the size of text for this message is the same as when regular content is displayed).
- Please add `font-style: italic;`.